src/components/LibraryComparison/index.tsx

```diff
@@ -57,7 +57,10 @@ export const LibraryComparison = () => {
     `)
 
     const renderAvailability = (isAvailable?: boolean) => {
-        return isAvailable ? <img className="w-4 h-4" src={CheckIcon} /> : <img className="w-4 h-4" src={XIcon} />
+        if (isAvailable === null || isAvailable === undefined) {
+            return null
+        }
```

Comment:
```suggestion
        if (isAvailable === null) {
            return null
        }
```

The loose equality (==) will check for both null and undefined at the same time. So isAvailable == null effectively checks if isAvailable is either null or undefined.